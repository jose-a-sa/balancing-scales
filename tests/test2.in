#ScaleName,Left,Right
A1,B1,B2
B1,4,B3
B2,B4,8
B3,2,6
B4,1,B5
B5,3,3
