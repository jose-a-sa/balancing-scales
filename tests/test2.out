A1,4,0
B1,9,0
B2,0,7
B3,4,0
B4,6,0
B5,0,0
